helix-term/tests/test/commands/variable_expansion.rs

```diff
@@ -0,0 +1,81 @@
+use super::*;
+use helix_view::editor::expand_variables;
```

Comment:
Integration tests shouldn't be calling functions like this directly. Instead these should use `test_key_sequence` with `:echo` and `Editor::get_status`